tests/integration/test_integratord/test_integratord_change_inode_alert.py

```diff
@@ -96,7 +96,7 @@ def test_integratord_change_json_inode(configuration, metadata, set_wazuh_config
             brief: Configure the local internal options file.
         - restart_wazuh_function:
             type: fixture
-            brief: Restart wazuh-modulesd daemon before starting a test, and stop it after finishing.
+            brief: Restart all daemons because Integratord depends on multiple daemons. Stop them after finishing.
```

Comment:
What are the multiple daemons you need?

Reply:
@damarisg as for now, I only consider necessary `wazuh-analisysd` and `wazuh-integratord`; so I will change the restart function to restart only those.

Reply:
I checked that, and It's correct!